### PR TITLE
Cloud storage

### LINKED ISSUE #2: Need to connect React to Cloud Storage
We need this to store resumes in here that are public so that we can see them from the browser.

I just realized that we are not going to be touching the API at all for this. We just need to figure out how to display the PDF to the user in a new tab

### PR DESCRIPTION
**Pull request by ynigoreyes**
I have it set up to automatically look for resume_1->4 on GCloudStorage using tags and `_blank` targets. I remember you telling me this haha. Good call

closes #2

### CODE REVIEW

**Comment by ynigoreyes** [comment]
Don't worry most of the changes are in yarn.lock lol

File: README.md [hunk 1]
```diff
@@ -28,24 +28,32 @@ Note: If you are using Go modules (i.e. `go mod`) to track your dependencies, se
...
+Optional: If you prefer not to use the root account to connect to the Cloud SQL instance, go back to the Cloud SQL console and select the `USERS` tab. Click `Create user account` and replace the username and password in the connection string. Make sure that `Allow any host(%)` is checked. Make sure the username and password are seperated by a colon in the connection string (`username:password`).
 
 ## Tips:
 - *How do I make data public?*
   - You have to add `Users-allUser-Reader` to the permissions of the document
+- *How do I connect to Cloud SQL locally?*
+	1. Use [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/quickstart-proxy-test)
+	2. Take note of the instance connection name of your target [Cloud SQL Instance](https://console.cloud.google.com/sql/instances) in the `Instance Connection Name` section
+	3. Install [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/quickstart-proxy-test) for your machine
+	4. Make sure that you do not have any process running on port 3306 by running `lsof -i tcp:3306`
```
Comment (swoldemi) [inline README.md]: Windows equivalent for item 4: `netstat -ano -p tcp` .

**Review by swoldemi — CHANGES_REQUESTED** [review]
May want to explain how to connect to the Cloud SQL instance using MySQL Workbench (i.e. create new connection, leave default IP and port, paste root password, click schemas tab to view tables, etc). Worked fine for me though!

**Comment by swoldemi** [comment]
Looks good (or at least I think so since a lot of the changes are React additions lol 👀). I was able to connect to the instance using the Cloud SQL Proxy and MySQL Workbench in less than 5 minutes. Requested 1 change in the README.

**Pull request merged**